cime_config/buildlib

```diff
@@ -31,29 +31,38 @@ def buildlib(caseroot, libroot, bldroot):
         fmsbuilddir = os.path.join(bldroot,"FMS")
         if not os.path.exists(fmsbuildlib):
             #todo: call checkout_externals to get this component
-            expect(False, "FMS external not found")
```

Comment:
I think I should go back to the `expect(False` method of raising an error... testing that now